### PR TITLE
Addition to Multitenancy considerations doc

### PR DESCRIPTION
**Pull request by writer-jill**
Small addition to Multitenancy considerations doc
This PR has:
- [x] been self-reviewed.

### CODE REVIEW

File: docs/querying/multitenancy.md [hunk 1]
```diff
@@ -75,10 +75,7 @@ stored on this tier.
 
 ## Supporting high query concurrency
 
-Druid's fundamental unit of computation is a [segment](../design/segments.md). Processes scan segments in parallel and a
-given process can scan `druid.processing.numThreads` concurrently. To
-process more data in parallel and increase performance, more cores can be added to a cluster. Druid segments
-should be sized such that any computation over any given segment should complete in at most 500ms.
+Druid's fundamental unit of computation is a [segment](../design/segments.md). Processes scan segments in parallel and a given process can scan `druid.processing.numThreads` concurrently. To process more data in parallel and increase performance, more cores can be added to a cluster. Druid segments should be sized such that any computation over any given segment should complete in at most 500ms. The metric you need to check for this is [`query/segment/time`](../operations/metrics.md#historical).
```
Comment (techdocsmith) [inline docs/querying/multitenancy.md:78]:
```suggestion
Druid's fundamental unit of computation is a [segment](../design/segments.md). Processes scan segments in parallel and a given process can scan `druid.processing.numThreads` concurrently. You can add more cores to a cluster to process more data in parallel and increase performance. Size your Druid segments such that any computation over any given segment should complete in at most 500ms. Use the  the [`query/segment/time`](../operations/metrics.md#historical) metric to monitor computation times.
```

**Review by techdocsmith — CHANGES_REQUESTED** [review]
minor changes for style

File: docs/querying/multitenancy.md (hunk 1 above)
Comment (kfaraz) [inline docs/querying/multitenancy.md:78]: Nit: are we using the possessive case with `Druid` elsewhere, @techdocsmith ?

Maybe we could fix this up too. e.g.
`In Druid, the fundamental unit of computation is a segment`
or
`Druid uses a segment as the fundamental unit of computation`
or something to that effect.

**Comment by jill-imply** [comment]
Made changes suggested by @kfaraz and @techdocsmith - ready for re-review.

**Review by techdocsmith — APPROVED** [review]
LGTM

**Review by kfaraz — APPROVED** [review]
LGTM

**Pull request merged**